packages/frontend/core/src/components/page-detail-editor.tsx

```diff
@@ -125,6 +103,25 @@ const EditorWrapper = memo(function EditorWrapper({
     [switchToEdgelessMode, switchToPageMode, pageId]
   );
 
+  //hack: scroll to block when jumping to block
+  const [elementPath, setElementPath] = useState<string[]>([]);
+  const blockElement = document.querySelector(
+    `[data-block-id="${blockId}"]`
+  ) as BlockElement | null;
```

Comment:
do not query dom element in render

Reply:
Sorry, I haven't found a way to make this function run normally without writing it like this.

Reply:
You can query it in `useEffect` when needed